plugins/inputs/openstack/openstack.go

```diff
@@ -71,7 +71,7 @@ type OpenStack struct {
 	Username         string          `toml:"username"`
 	Password         string          `toml:"password"`
 	EnabledServices  []string        `toml:"enabled_services"`
-	ServerDiagnotics bool            `toml:"server_diagnotics"`
+	ServerDiagnostics bool            `toml:"server_diagnostics"`
```

Comment:
This will break existing configs. Please deprecate the old option and add a new one...